### PR TITLE
CASMCMS-7647: add change of NCN user password location

### PR DESCRIPTION
**Pull request by rkleinman-hpe**
## Summary and Scope

Add notes to the NCN user and ssh keys change procedures about the
change of location in CSM 1.2 of the user passwords on NCNs.

## Issues and Related PRs

* Resolves CASMCMS-7647

## Testing

N/A

## Risks and Mitigations

If users do not follow this advice, their root password changes from the 1.0 setting will not be applied to the NCN nodes properly during NCN customization.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct

### CODE REVIEW

File: operations/security_and_authentication/SSH_Keys.md [hunk 1]
```diff
@@ -71,12 +63,17 @@ personalization.
    the values in the Ansible role. See `roles/csm.ssh_keys/README.md` in the
    repository for more information.
 
+### Procedure: Apply Root SSH Keys to NCNs (Standalone)
+
+Use the following procedure with the `rotate-ssh-keys-mgmt-nodes.yml` playbook to
+**only** change the root ssh keys on NCNs. This is a quick alternative to
```
Comment (nrockershousen) [inline operations/security_and_authentication/SSH_Keys.md:69]: ssh -> SSH

File: operations/security_and_authentication/Update_NCN_Passwords.md [hunk 2]
```diff
@@ -12,6 +12,14 @@ instance, and applied with the `csm.password` Ansible role via a CFS session. If
 no password is added to Vault as in the procedure below, this Ansible role will
 skip any password updates.
 
+### New in CSM Release 1.2.0
+
+> The location of the password secret in Vault has changed in CSM version 1.2. The
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_NCN_Passwords.md:17]: The > characters can be removed

File: operations/security_and_authentication/Update_NCN_Passwords.md [hunk 3]
```diff
@@ -12,6 +12,14 @@ instance, and applied with the `csm.password` Ansible role via a CFS session. If
...
+> previous location (`secret/csm/management_nodes root_password=...`) has been
+> changed to `secret/csm/users/root password=...`. You must set the password in
+> the new location using the _Configure Root Password in Vault_ procedure below
+> for it to be applied to the NCN nodes.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_NCN_Passwords.md:21]: Remove redundant use of "nodes" and change NCN to NCNs.

File: operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md [hunk 4]
```diff
@@ -49,44 +49,50 @@ files.
...
+**NOTE**: This key pair may or may not be the same key pair used for the NCN
+`root` user. See the _Configure the Root Password and Root SSH Keys in Vault_
+procedure below for setting the root user SSH keys on NCNs.
+
 ###  Option 3: Disable CSM-provided Passwordless SSH
 
 Local site security requirements may preclude use of passwordless SSH access between
 management nodes. A variable has been added to the associated ansible roles that will
-allow you to disable passwordless ssh setup to any or all nodes.
+allow you to disable passwordless ssh setup to any or all nodes. From the cloned
```
Comment (nrockershousen) [inline operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md:84]: ssh -> SSH

File: introduction/differences.md [hunk 5]
```diff
@@ -33,6 +33,8 @@ The most noteworthy changes since the previous release are described here.
       * Introduces the cray-dns-powerdns, cray-dns-powerdns-postgres, and cray-powerdns-manager pods
       * Supports zone transfer to external DNS servers via AXFR query and DNSSEC
       * Please refer to the [DNS overview](../operations/network/dns/DNS.md) and [PowerDNS Configuration Guide](../operations/network/dns/PowerDNS_Configuration.md) for further information.
+   * NCN User password and SSH key management is available for both root and non-root users via NCN personalization. Please refer to
```
Comment (nrockershousen) [inline introduction/differences.md:36]: User -> user

**Review by nrockershousen — APPROVED** [review]

**Review by phalseth-hpe — APPROVED** [review]

**Pull request merged**